### ISSUE TITLE
loss中的self.reparameterize这个方法没有

### ISSUE THREAD

**Issue by ZhuangQingYuan** [Haoyu-Xie/PRC2#1]
generalized_negatives = torch.cat([self.reparameterize(mmt_prototype_mu[negative_id], mmt_prototype_sigma[negative_id])], dim=1)
